mobify/platform-scaffold
# Update AMP SDK - 0.3.1

**JIRA**: https://mobify.atlassian.net/browse/AMP-88

## Changes
- Update `mobify-amp-sdk` to `0.3.1`
- Change `AmpImage` to `Img`
- Fix `Search` and `Lightbox` path to AMP SDK

## How to test-drive this PR
- npm install
- Run `npm run dev`
- Preview [Merlin's Potions](http://localhost:3000/potions.html)
- Click on Search button in header and make sure that lightbox and search are working.
